subcommands/keys/tuf_updates_review.go

```diff
@@ -56,9 +61,37 @@ func doTufUpdatesReview(cmd *cobra.Command, args []string) {
 		} else {
 			rootToShow = newCiRoot
 		}
-		bytes, err := subcommands.MarshalIndent(rootToShow, "", "  ")
-		subcommands.DieNotNil(err)
-		fmt.Println(string(bytes))
+
+		if showRaw {
+			bytes, err := subcommands.MarshalIndent(rootToShow, "", "  ")
+			subcommands.DieNotNil(err)
+			fmt.Println(string(bytes))
+		} else {
+			var baseRootToShow *client.AtsTufRoot
+			if showProd {
+				if updates.Current.ProdRoot != "" {
+					subcommands.DieNotNil(
+						json.Unmarshal([]byte(updates.Current.ProdRoot), &baseRootToShow),
+						"Current prod root",
+					)
+				} else {
+					// First rotation, old prod root equals old CI root
+					baseRootToShow = oldCiRoot
+				}
+			} else {
+				baseRootToShow = oldCiRoot
+			}
+
+			before, err := subcommands.MarshalIndent(baseRootToShow, "", "  ")
+			subcommands.DieNotNil(err)
+			after, err := subcommands.MarshalIndent(rootToShow, "", "  ")
+			subcommands.DieNotNil(err)
+			diff := godiff.Strings(
+				strings.Split(string(before), "\n"),
+				strings.Split(string(after), "\n"),
+			)
+			fmt.Println(strings.Join(diff, "\n"))
+		}
```

Comment:
We already have the `github.com/fatih/color` module. You could probably look at `line[0] == '+'` and make them green and `line[0] == '-'` and make them red.

Reply:
Makes sense. I just wanted to keep the scope manageable; so waited for someone to make a comment on this.

Reply:
https://github.com/foundriesio/fioctl/pull/211/commits/57b0f5532cbd1efaecb77649ef119aa7cded93ce